isis/src/newhorizons/apps/lorri2isis/tsts/summing4/Makefile

```diff
@@ -1,12 +0,0 @@
-APPNAME = lorri2isis
-
-include $(ISISROOT)/make/isismake.tsts
-
-commands:
-	$(APPNAME) from=$(INPUT)/lor_0030710290_0x633_sci_1.fit \
-	  to=$(OUTPUT)/lor_0030710290_0x633_sci_1.cub \
-	  > /dev/null;
-	catlab from=$(OUTPUT)/lor_0030710290_0x633_sci_1.cub \
-	  to=$(OUTPUT)/lor_0030710290_0x633_sci_1.pvl > /dev/null;
```

Comment:
Any particular reason that this didn't get converted? I would assume because the summing modes don't effect the import app and the main logic for getting the summing mode is in the translation.

Reply:
@acpaquette I was not sure how this test was different than the tests I already implemented. If you think it'll be useful to add it in, let me know.

Reply:
I think this was testing a translation file specific look up. If you checkout [this translation file](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/isis/src/newhorizons/apps/lorri2isis/NewHorizonsLorriInstrument_fit.trn) the it looks for a particular key for the summing mode. 

If the app was doing anything with the summing mode I would suggest reimplementing the test as a gtest. As you said this test isn't doing anything directly with the app so I think we can keep it as is. Good job on this PR!